SAMtags.tex

```diff
@@ -453,6 +453,165 @@ \subsubsection{Color space}
 Color read quality on the original strand of the read. Same encoding as {\sf QUAL}; same length as {\tt CS}.
 \end{description}
 
+\section{Draft tags}
+
+These are tags which have been proposed and are broadly accepted to
+become standard tags, but a review or probationary period has been
+deemed useful.  They use the locally-defined tag namespace and
+processing software should consider that the tags may have local usage
+for other purposes.
+
+\begin{center}\small
+% This table is sorted alphabetically
+\begin{longtable}{ccp{12.5cm}}
+  \hline
+  {\bf Tag} & {\bf Type} & {\bf Description} \\
+  \hline
+  \endhead
+  {\tt Mm} & i & Base modifications / methylation  \\
+% {\tt MP} & i & Base modification qualities \\
+  {\tt Ml} & i & Base modification probabilities \\
+\end{longtable}
+\end{center}
+
+\subsection{Base modifications}
+
+Base modifications, including base methylation, are represented as a series of edits from the primary unmodified sequence as originally reported by the sequencing instrument.
+This potentially differs to the sequence stored in the main SAM {\sf SEQ} field if the latter has been reverse complemented, in which case SAM {\sf FLAG} 0x10 must be set.
+This means modification positions are also recorded against the original orientation (i.e. starting at the 5' end), and count the original base types.
+
+Each modified base listed also has a quality value associated with it.
+Given the unmodified base already has a phred likelihood, this base modification quality should be interpreted as the likelihood of this modification being correct given an assumption the original call is correct.
+
+\begin{description}
+\item[Mm:Z:\tagregex{([ACGTUN][-+][a-z]+(,[0-9]+)*;)*}]
+\hfill\\
+The first character is the unmodified ``fundamental'' base as reported
+by the sequencing instrument.
```

Comment:
what if SEQ has IUPAC codes? how does the counting work then?

Reply:
They'd be covered as N as it explicitly said "N for anything else".  I'll make that more detailed with an explicit clause to cover IUPAC codes.

Reply:
Note some people use the phrase "canonical base" instead of "fundamental base".  I'm happy with either - votes either way welcomed.